ui/src/components/homePage/homePage.ts

```diff
@@ -1,12 +1,38 @@
 import { Container } from '@Core/container';
 import { Header } from '@Components/homePage/homePage/header';
+import { Login } from '@Components/homePage/homePage/popups/login';
+import { Registration } from '@Components/homePage/homePage/popups/registration';
+import { Welcome } from '@Components/homePage/homePage/popups/welcome';
 
 export class HomePage extends Container {
-    protected LOCATORS = {
+    private LOCATORS = {
         header: this.page.locator('//div[contains(@class, "header__wrapHeader")]'),
+        loginPopup: this.page.locator('//div[contains(@class, "loginPopup") and ./section[not(contains(@class, "createAccount"))]]'),
+        registrationPopup: this.page.locator('//div[contains(@class, "loginPopup") and ./section[contains(@class, "createAccount")]]'),
+        welcomePopup: this.page.locator('//div[contains(@class, "welcomePopup__wrapper")]'),
     };
 
     public Header = new Header(this.LOCATORS.header, this.page);
+    public Login = new Login(this.LOCATORS.loginPopup, this.page);
+    public Registration = new Registration(this.LOCATORS.registrationPopup, this.page);
+    public Welcome = new Welcome(this.LOCATORS.welcomePopup, this.page);
```

Comment:
Better naming for it will be Login -> LoginPopup, Registration -> RegistrationPopup, Welcome -> WelcomePopup